.pytool/Plugin/DscCompleteCheck/Readme.md

```diff
@@ -29,4 +29,4 @@ Path to DSC to consider platform dsc
 
 ### IgnoreInf
 
-Ignore error if Inf file is not listed in DSC file
+A list of paths in git ignore syntax to ignore in the check. These can include directory and file paths.

```

Comment:
It may be useful to specify the git ignore syntax is not relative to the workspace root, but to the package path

Reply:
Updated